cms/static_src/javascript/auth.js

```diff
@@ -0,0 +1,153 @@
+/* eslint-disable no-console */
+
+import SessionManagement from 'dis-authorisation-client-js';
+
+/** ----------------------------------------------------------------
+ *  Grab the JSON produced by:
+ *      {{ AUTH_CONFIG | json_script('auth-config') }}
+ *  in the Jinja template (or the Python hook).
+ * ----------------------------------------------------------------*/
+
+let authConfig = {};
+const configEl = document.getElementById('auth-config');
+
+if (configEl) {
+  try {
+    authConfig = JSON.parse(configEl.textContent);
+  } catch (e) {
+    console.error('[WAGTAIL] Failed to parse <script id="auth-config">:', e);
+  }
+}
+
+const {
+  wagtailAdminHomePath,
+  csrfCookieName,
+  csrfHeaderName,
```

Comment:
Suggestion: With the below, `csrfCookieName` can be removed. `csrfHeaderName` can also be obtained using `window.wagtailConfig.CSRF_HEADER_NAME `